src/test/java/org/fisco/bcos/channel/test/parallel/precompile/transferSign.java

```diff
@@ -0,0 +1,61 @@
+package org.fisco.bcos.channel.test.parallel.precompile;
+
+import java.math.BigInteger;
+import org.fisco.bcos.web3j.crypto.EncryptType;
+import org.fisco.bcos.web3j.crypto.gm.sm2.SM2Sign;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class transferSign {
+    private static Logger logger = LoggerFactory.getLogger(transferSign.class);
+
+    public static void Usage() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'Usage' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)